crates/turbopack-ecmascript/src/transform/mod.rs

```diff
@@ -77,8 +78,10 @@ pub enum EcmascriptInputTransform {
 
 /// The CustomTransformer trait allows you to implement your own custom SWC
 /// transformer to run over all ECMAScript files imported in the graph.
+#[async_trait]
 pub trait CustomTransformer: Debug {
-    fn transform(&self, program: &mut Program, ctx: &TransformContext<'_>) -> Option<Program>;
+    async fn transform(&self, program: &mut Program, ctx: &TransformContext<'_>)
+        -> Option<Program>;
```

Comment:
Why does it return an `Option<Program>` when it already has a `&mut Program`?

It looks like that `Option<Program>` is not really needed.